### PR TITLE
KTOR example

### PR DESCRIPTION
**Pull request by arolfes**

### CODE REVIEW

File: ktor-sample/src/main/kotlin/novatec/info/model/DatabaseFactory.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import org.jetbrains.exposed.sql.SchemaUtils.create
+import org.jetbrains.exposed.sql.insert
+import kotlin.random.Random
+
+
+object DatabaseFactory {
+    fun init() {
+        Database.connect(hikari())
+        transaction {
+            create(BooksTable, AuthorsTable)
```
Comment (carlosbarragan) [inline ktor-sample/src/main/kotlin/novatec/info/model/DatabaseFactory.kt]: Is there a way to use Flyway for data migration? I would like that the applications are as similar as possible.

File: ktor-sample/src/main/kotlin/novatec/info/plugins/Routing.kt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import io.ktor.server.routing.*
+import io.ktor.server.response.*
+import io.ktor.server.application.*
+import novatec.info.model.AuthorRepository
+
+fun Application.configureRouting(authorRepository: AuthorRepository) {
+    routing {
+        get("/authors") {
+            val authors = authorRepository.getAllAuthors()
+            println(authors)
```
Comment (carlosbarragan) [inline ktor-sample/src/main/kotlin/novatec/info/plugins/Routing.kt]: I think this `println()` slipped without being noticed 😉

**Review by carlosbarragan — CHANGES_REQUESTED** [review]

File: ktor-sample/src/main/kotlin/novatec/info/model/DatabaseFactory.kt (hunk 1 above)
Comment (arolfes) [reply ktor-sample/src/main/kotlin/novatec/info/model/DatabaseFactory.kt]: yes it is, like you did in the minimal example

File: ktor-sample/src/main/kotlin/novatec/info/plugins/Routing.kt (hunk 2 above)
Comment (arolfes) [reply ktor-sample/src/main/kotlin/novatec/info/plugins/Routing.kt]: exactly
thanks for finding

**Pull request merged**